internal/cloud/realm/secrets.go

```diff
@@ -37,3 +37,25 @@ func (c *client) Secrets(groupID, appID string) ([]Secret, error) {
 	}
 	return secrets, nil
 }
+
+type createSecretRequest struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+func (c *client) CreateSecret(groupID, appID, name, value string) error {
```

Comment:
Thoughts on having some sort of e2e test here for secrets? Once we have secrets delete we can probably do something like:
1. Create App
2. Test secrets create
3. Test secrets list
4. test secrets delete
5. Delete app

Reply:
yes, you should add that test in this PR, at least the parts of creating and listing.  Check out `realm/user_ext_test.go` for examples on these types of tests